ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -399,16 +403,16 @@ export default defineComponent({
       // When not a draft Transfer, force no unsaved changes after loading current owners
       !getMhrInformation.value.draftNumber && await setUnsavedChanges(false)
 
-      localState.accountInfo = await getSubmittingPartyInformation()
+      localState.accountInfo = await getAccountInformation()
       parseSubmittingPartyInfo()
       localState.loading = false
 
       localState.dataLoaded = true
     })
 
     // Get Account Info from Auth to be used in Submitting Party section in Review screen
-    const getSubmittingPartyInformation = async (): Promise<AccountInfoIF> => {
-      return getAccountInfoFromAuth()
+    const getAccountInformation = async (): Promise<AccountInfoIF> => {
+      return isRoleQualifiedSupplier.value === true ? getAccountInfoFromAuth() : {} as AccountInfoIF
```

Comment:
Condition can be simplified to `!!isRoleQualifiedSupplier.value`

So for staff in the this instance, would they not want to pull in the submitting party from the Reg?
I'm assuming this is a separate (future) ticket?

Reply:
See comment below..

Reply:
Changed line 415.